src/App.tsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [movies, setMovie] = useState([...moviesFromServer]);
+  const [input, setInput] = useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setInput(value);
+
+    const filteredMovies = moviesFromServer.filter(
+      movie => movie.title.toLowerCase().includes(value.toLowerCase().trim())
+      || movie.description.toLowerCase().includes(value.toLowerCase().trim()),
+    );
+
+    setMovie(filteredMovies);
```

Comment:
you don't need `movies` state - filtered movies will change after every `input` change anyway because it's saved in `state` i.e cause re-render after every change

Reply:
Thanks for advise :)